helmfile.d/values/velero-wc.yaml.gotmpl

```
@@ -44,10 +60,13 @@ initContainers:
         name: plugins
   {{- end }}
 
-snapshotsEnabled: false
+snapshotsEnabled: {{ .Values.velero.useVolumeSnapshots }}
 
 configuration:
   defaultVolumesToFsBackup: true
+  {{- if .Values.velero.useVolumeSnapshots }}
+  features: "EnableCSI"
+  {{- end }}
 
   uploaderType: {{ .Values.velero.uploaderType }}  # default is kopia
```

Nit, could you remove this comment which is something I added before, since the default is still restic
```suggestion
  uploaderType: {{ .Values.velero.uploaderType }}
```